pkg/backplane/status/status.go

```diff
@@ -0,0 +1,45 @@
+package status
```

Comment:
Why not call it update?  

e.g. - update.Status

Reply:
I think `update` is a bit too vague. Update what? Could be status, plugin versions, kubernetes versions...

Reply:
This is all Run / Job Specific , but I'm also terrible at naming and usually redux it over time.  I don't have strong opinions so whatever makes sense for now is fine with a //TODO - Find a better name...

Reply:
I was reading through some go docs and I think I was wrong to say `status.Status` is bad. The stutter I was confusing it with would be something like `status.NewStatus`. I'm ok with `status.Status` as an exported type.